ab-testing/scripts/deploy/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const mvtIdsDictionaryName = 'mvt_ids';
+export const abTestIdsDictionaryName = 'ab_test_ids';
```

Comment:
Doesn't look like these are used anywhere (yet)?

Reply:
Yes thanks! Don't need the dictionary names, use the ids from environment variables!